WordPress/build.gradle

```diff
@@ -120,9 +122,18 @@ dependencies {
     compile 'org.wordpress:slidinguppanel:1.0.0' // not found in maven central
     compile 'org.wordpress:passcodelock:1.2.0'
 
+    // Dagger
+    compile 'com.google.dagger:dagger:2.0.2'
+    apt 'com.google.dagger:dagger-compiler:2.0.2'
+    provided 'org.glassfish:javax.annotation:10.0-b28'
+
     // Simperium
     compile 'com.simperium.android:simperium:0.6.8'
 
+    compile ('com.github.wordpress-mobile:WordPress-Stores-Android:tmp-test-max-SNAPSHOT') {
```

Comment:
This is a temporary branch, I'm waiting for some PRs to be merged in wpstores before changing that to `develop-SNAPSHOT`